src/main/java/com/bieganski/ox/PositionCalculator.java

```diff
@@ -11,7 +11,7 @@
 //TODO maybe parser?
     int calculatePosition(String userInput) {
         if (!positionValidator.checkString(userInput))
-            throw new IllegalArgumentException("There is no such position!");
+            return - 1;
```

Comment:
Zwykle taka zmiana jest wspominana w dokumentacji. Było nie było, -1 jako szczególna wartość zasługuje na słowo. Zwłaszcza jak poprzednio leciał wyjątek z komentarzem.

Reply:
Dodałem javadok z opisem co to znaczy.